packages/nartus_geocoder/lib/src/geocoder_service.dart

```diff
@@ -0,0 +1,13 @@
+part of "../nartus_geocoder.dart";
+
+class GeocoderService {
+  Future<Placemark?> getCurrentPlaceCoding(
+      double latitude, double longitude) async {
+    List<Placemark> placemarks = await GeocodingPlatform.instance
```

Comment:
instead of returning a `List<Placemark>`, you can return an object which has `String? businessName`, and `String? address`